net-vpn/i2pd/files/i2pd-2.25.0-lib-path.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/build/CMakeLists.txt b/build/CMakeLists.txt
+index 949f6a46..42165524 100644
+--- a/build/CMakeLists.txt
++++ b/build/CMakeLists.txt
+@@ -103,8 +103,8 @@ set_target_properties(libi2pd PROPERTIES PREFIX "")
+ if (WITH_LIBRARY)
+   install(TARGETS libi2pd
+     EXPORT libi2pd
+-    ARCHIVE DESTINATION lib
+-    LIBRARY DESTINATION lib
++    #ARCHIVE DESTINATION lib
```

Comment:
Don't comment things out in a patch-hack. Just remove them altogether.

Also, have you reported the problem upstream?

Reply:
Deleted. Did not report, I don't yet understand myself what should be there instead.